frontend/views/containers/GroupPledgesGraph.vue

```diff
@@ -8,25 +8,25 @@
     span.is-title-4 {{ withCurrency(graphData.groupGoal) }}
 
   ul.c-legendList(:aria-label='L("Group pledging summary")' data-test='groupPledgeSummary')
-    graph-legend-item(
+    graph-legend-item.c-wlegend-side(
```

Comment:
When I saw this class I looked for it at `<styles>` below and didn't find it. I was about to comment that this wasn't needed. Then, I realized this class is declared inside `GraphLegendItem` component.

I suggest adding a prop `variant` to the component, then it would handle `c-wlegend-side` based on its value. And here we pass `variant='side'`. Similar to how we do it with `color` prop.
The default variant value can be `inline` for the Overview widget.

Another advantage of using approach is [avoiding impossible states](https://kentcdodds.com/blog/make-impossible-states-impossible). In this case I could pass `c-wlegend-side.c-wlegend-inline` and it would create something weird.